sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -892,3 +898,186 @@ def test_fit_predict(self, estimator_instance, n_columns):
         cutoff = get_cutoff(y_train, return_index=True)
         _assert_correct_pred_time_index(y_pred.index, cutoff, fh)
         _assert_correct_columns(y_pred, y_train)
+
+
+class TestAllGlobalForecasters(TestAllObjects):
+    """Module level tests for all global forecasters."""
+
+    estimator_type_filter = "global_forecaster"
+
+    def test_global_forecasting_tag(self, estimator_class):
+        global_forecasting_tag = estimator_class._tags["capability:global_forecasting"]
+        assert global_forecasting_tag is True
+
+    def test_pridect_signature(self, estimator_class):
+        sig = signature(estimator_class.predict)
+        assert "X" in sig.parameters.keys()
+        assert "y" in sig.parameters.keys()
+
+    def test_global_forecasting_multiindex_hier(self, estimator_instance):
+        from sktime.utils._testing.hierarchical import _make_hierarchical
+
+        data_length = 100
+        data = _make_hierarchical(
+            (5, 100),
+            n_columns=2,
+            max_timepoints=data_length,
+            min_timepoints=data_length,
+        )
+        X_train = data["c0"].to_frame()
+        y_train = data["c1"].to_frame()
+
+        max_prediction_length = 3
+        fh = ForecastingHorizon(range(1, max_prediction_length + 1), is_relative=True)
+
+        estimator_instance.fit(y_train, X_train, fh=fh)
+
+        X_test = X_train
+        # remove max_prediction_length from the end of y_train
+        len_levels = len(y_train.index.names)
+        y_test = y_train.groupby(level=list(range(len_levels - 1))).apply(
+            lambda x: x.droplevel(list(range(len_levels - 1))).iloc[
+                :-max_prediction_length
+            ]
+        )
+        y_pred = estimator_instance.predict(fh, X_test, y_test)
+
+        # TODO
```

Comment:
Here is a TODO?

Reply:
`_assert_correct_pred_time_index` does support multiindex. I will simply pass the index of the first instance for now.